pychromecast/dial.py

```diff
@@ -90,7 +90,7 @@ def _get_status(
             _LOGGER.debug("Resolved service %s to %s", service, host)
             break
 
-    headers = {"content-type": "application/json"}
+    headers = {"host": "", "content-type": "application/json"}
```

Comment:
The purpose of setting host to "" needs to be explained in a comment.
It's mentioned in the PR description that:
> I noticed that my NVIDIA Shield blocks any request (empty body with status code 403) when a domain is set in the header, at least when requesting via IPv6

Does that mean the request is *not* blocked if requesting via IPv4? What is the host header set to in case of requesting the IPv4 and IPv6?

Reply:
Added a comment.

I've now also tested this with IPv4 and a domain, and it's blocked too, so it's not IPv6 specific. In addition to the Shield, I tested it with a Google Home Mini, which shows the same behaviour.

The default behaviour with a plain IP is also to send it as the Host header, just like with a domain. Technically, not sending a Host header is [against the spec](https://www.rfc-editor.org/rfc/rfc7230#section-5.4), but so is accepting a request that doesn't have one, which Chromecast devices currently do. To be within spec, any domain would have to be resolved first, and the request would have to be made against the raw IP.

Reply:
I see. Then please move this change to a separate PR since it's not related to IPv6 support.